lib/src/third_party/diff_match_patch/README.md

```diff
@@ -0,0 +1,12 @@
+# Diff-Match-Patch
```

Comment:
Any reason not to use https://pub.dev/packages/diff_match_patch ?

Reply:
We just needed the diff functionality and thought it was  better not to depend on third party library.

Reply:
It is really strange to showcase its license and tests in the middle of the pana package. I'd rather have it as a dependency. wdyt @jonasfj ?